react-app/src/Finish/index.js

```diff
@@ -37,24 +37,26 @@ export default function Finish({ db, setReset }) {
   }, [])
 
   return (
-    <PageTemplate topBarColor={ThemeColor.background} bottomColor={ThemeColor.background}>
-      <Header />
-      <Info
-        moves={exercise.length}
-        // These coming two lines are kinda hard code
-        time={id == 1 ? 12 : 14}
-        format={`Sitting`}
-      ></Info>
-      <Image />
+    <>
       <Lottie
         source={require('../../assets/animation/62717-confetti.json')}
         autoPlay
         loop
         style={styles.animation}
         resizeMode='cover'
       />
-      <Buttons id={id} title={title} />
-    </PageTemplate>
+      <PageTemplate topBarColor={ThemeColor.background} bottomColor={ThemeColor.background}>
+        <Header />
+        <Info
+          moves={exercise.length}
+          // These coming two lines are kinda hard code
+          time={id == 1 ? 12 : 14}
```

Comment:
We can change it to a function calculating total duration regardless of the exercise type

Reply:
I will create a ticket for optimizing the data structure. So I won't change this hard code for now.